SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizer.cc

```diff
@@ -123,21 +120,14 @@ namespace cms
 	}
       }
     }
-  
-    // one type of Digi and DigiSimLink suffices 
-    // changes in future: InnerPixel -> Tracker
-    // creating algorithm objects and pushing them into the map
-    algomap_[AlgorithmType::InnerPixel] = std::unique_ptr<Phase2TrackerDigitizerAlgorithm>(new PixelDigitizerAlgorithm(iconfig_, (*rndEngine_)));
-    algomap_[AlgorithmType::PixelinPS]  = std::unique_ptr<Phase2TrackerDigitizerAlgorithm>(new PSPDigitizerAlgorithm(iconfig_, (*rndEngine_)));
-    algomap_[AlgorithmType::StripinPS]  = std::unique_ptr<Phase2TrackerDigitizerAlgorithm>(new PSSDigitizerAlgorithm(iconfig_, (*rndEngine_)));
-    algomap_[AlgorithmType::TwoStrip]   = std::unique_ptr<Phase2TrackerDigitizerAlgorithm>(new SSDigitizerAlgorithm(iconfig_, (*rndEngine_)));
   }
-
   void Phase2TrackerDigitizer::endLuminosityBlock(edm::LuminosityBlock const& lumi, edm::EventSetup const& iSetup) {
-    algomap_.clear();
+    edm::LogInfo("Phase2TrackerDigitizer") << "End Luminosity Block";
   }
+
   Phase2TrackerDigitizer::~Phase2TrackerDigitizer() {  
     edm::LogInfo("Phase2TrackerDigitizer") << "Destroying the Digitizer";
+    algomap_.clear();
```

Comment:
Calling `clear()` is unnecessary (`algomap_`'s destructor will clean up all its contents automatically).